### PR TITLE
Use "model" not "version" as the noun

### PR DESCRIPTION
**Pull request by erbridge**
We specify a model by the pattern `owner/model@version`.

Conceptually, the model is the thing, not the version. The version is a way of specifying which version of the model we want, and doesn't make much sense in isolation (even if the API itself doesn't ask for the model (yet)).

### CODE REVIEW

**Comment by bfirsh** [comment]
Hmm. We don't use this noun anywhere else. The `replicate.version()` API doesn't sit well either, but maybe I'm missing the thinking here. Let's maybe hash this out live?

**Comment by erbridge** [comment]
> We don't use this noun anywhere else.

We do in our docs: https://replicate.com/docs/reference/http#get-model-versions

File: lib/ReplicateClient.js [hunk 1]
```diff
@@ -21,8 +21,68 @@ export default class ReplicateClient {
...
-  version(id) {
-    return new Version({ id }, this);
+  async getModelVersions(nameOrParams) {
+    let owner, name;
+
+    if (typeof nameOrParams === "string") {
+      const nameComponents = /^([\w-]+)\/([\w-]+)$/.exec(nameOrParams);
+
+      if (!nameComponents) {
+        throw new ReplicateError(`Invalid name: ${nameOrParams}`);
```
Comment (zeke) [inline lib/ReplicateClient.js:31]:
```suggestion
        throw new ReplicateError(`Invalid model name: ${nameOrParams}. This should be a string in the format {owner}/{model}@{version} or an object with owner, model, and version strings.`);
```

**Review by zeke — APPROVED** [review]
I like it!

**Comment by nickstenning** [comment]
I started a discussion in Discord about whether this should be `owner/model@version` or `owner/model:version`, which we're already using in a handful of places: https://discord.com/channels/775512803439280149/887033123555115088/1078064767253618699

**Pull request merged**